_sips/minutes/2017-12-06-sip-minutes.md

```diff
@@ -0,0 +1,114 @@
+
+a) [SIP-33: Priority-based infix type precedence rules](http://docs.scala-lang.org/sips/make-types-behave-like-expressions.html)
+
+b) [SIP-NN: Adding prefix types](http://docs.scala-lang.org/sips/adding-prefix-types.html)
+
+**Seth** asks about the implementation status in Dotty and if there are any crucial differences in Scala 2 and Dotty?
+**Martin** and **Sebastien** agree there are none in regards to this SIP.
+The members are all in favour for this change and proceed to voting.
+
+**Conclusion** : The SIP-33 is accepted by unanimity.
+
+
+### [SIP-NN: Adding prefix types](http://docs.scala-lang.org/sips/adding-prefix-types.html)
+[YouTube time: 25'00 until the end](https://youtu.be/Mhwf15gjL9s?t=1503)
+
+**Jorge** introduces the SIP's development, based on the idea
+(in **Oron's** words) "it is easier to reason about the language when mathematical and logical operations for both terms and types are expressed the same"; goes over the motivation examples **Oron** proposed since the last SIP (*splice prefix types for meta programing; singleton-ops library and DFiant library example*) and opens the discussion about the recent use-cases.
+
+**Martin** starts with by introducing his PR, the use-case in Dotty, ["Principled Meta Programming"](https://gist.github.com/odersky/f91362f6d9c58cc1db53f3f443311140). **Sébastien** argues that **Martin's** use-case is not really related to what the SIP-NN is aiming to achieve, but in this context he would rather agree on special-casing the ˜ for macros and splices.
+
+However, **Sebastian** gives his preference to the SIP itself.
+**Martin**, on the other hand, is "dubious" about the SIP, stating that in Scala those 4 operators were defined originally because the syntax in Java. He disagrees with now making a step even further - elevating them to the principal. He is also sceptical because he foresees the "end-operator misuse" to which **Adriaan** adds the issue between *annotations* and *variants* that could make the confusion even deeper. 
+**Seth** and **Eugene** agree it could be too confusing, and even though there is a potential in unifying the language features, this particular SIP doesn't seem to address it in a clear and persuasive way.
+
+**Miles** proposes to let this SIP have its implementation in Typelevel Scala. He believes the arguments raised in this discussion could be tested and eventually even answered or "shaped" by the user's experience. He asks to defer the discussion until the use-case is ready. **Adriaan** supports the idea but underlines that it is important to know that implementation should not be considered as a guarantee leading to be a part of the language.
+
+The Committee proceeds with voting on numbering the SIP.
+
+**Conclusion**: The SIP-NN is numbered, from now SIP-36, it will be discussed once results of the Typelevel implementation are ready.
+
```

Comment:
Can we add a title here? Something like extra

Reply:
Addressed, good pint, thanks!